### DISCUSSION TITLE
Path.path attribute does not update after moving

### DISCUSSION THREAD

**Discussion by abdussami-tayyab** [fabricjs/fabric.js#8872] [Q&A]
### CheckList

- [X] I agree to follow this project's Code of Conduct
- [X] I have read and followed the [Contributing Guide](https://github.com/fabricjs/fabric.js/blob/master/CONTRIBUTING.md)
- [X] I have read and followed the [Issue Tracker Guide](https://github.com/fabricjs/fabric.js/blob/master/CONTRIBUTING.md#%EF%B8%8F-issue-tracker)
- [X] I have searched and referenced existing issues and discussions
- [x] I am filing a **BUG** report.
- [X] I have managed to reproduce the bug **after** upgrading to the **latest** version
- [X] I have created an accurate and minimal reproduction

### Version

5.3.0

### In What environments are you experiencing the problem?

Chrome

### Node Version (if applicable)

None

### Link To Reproduction

https://jsfiddle.net/9nor3bhv/1/

### Steps To Reproduce

1. Just try moving the Path object
2. Check the value of path which never changes

### Expected Behavior

The `fabric.Path` object should provide a start and an endpoint of the Path object because the bounding-box, or the top-left, or the `aCoords`/`oCoords` only gives the bounding-box attributes.

The expected behaviour for me is that when the Path object moves, or scales, I set a circle on both ends of the Path.

### Actual Behavior

The Path object does not update it's path values i.e. the path object OR the start and end of the Path

**Comment by ShaMan123** [comment]
This is intentional
Otherwise it is bad perf
Look at pathOffset and calc what you need

**Comment by abdussami-tayyab** [comment]
Can you share an example moving a Path object and how we can use pathOffset
and calc to know the updated Path?

Perhaps we can add an example to the documentation then.

**Comment by Synergism612** [comment]


**Reply by asturur** [reply]
Please add a machine translation for the other readers even if you are not comfortable with english.
